### ISSUE TITLE
Images downloaded using custom rules are not valid .jpg files: `Decode error! Not a JPEG file: starts with 0xc3 0xbf` (Chrome w/ TM beta only) 

### ISSUE THREAD

**Issue by darkred** [tophf/mpiv#120]
#### Steps to reproduce the bug
1. In a fresh 125.0.6422.61 Chrome profile, enable Developer mode in chrome://extensions/ and install latest Tampermonkey beta 5.2.6195, and latest MPIV.
2. Add the following custom rule:
    ```json
    {"d": "bandcamp.com",
     "r": "bcbits\\.com/img/.+?_(\\d{1,2})",
     "s": "/_(\\d{1,2})/_10/",
     "c": "return node.alt.replace(' thumbnail','')",
     "css": ".tooltip, .collected-by .deets .pic > .tooltip {visibility:hidden;opacity:0}"
    }
    ```
3. Visit e.g. https://paulgilbert.bandcamp.com/ , hover on "The Dio Album" cover thumbnail and press <kbd>d</kbd> (download)
4. Notice that the downloaded file is not a valid .jpg file: trying to open it with IrfanView results in
`Decode error! Not a JPEG file: starts with 0xc3 0xbf` (it's 686 KB -see attached .zip-) 

*Note that with Chrome+VM beta as well as with Firefox+TM beta the files are downloaded ok.*

#### Expected behavior
The downloaded file should be valid .jpg (in this case it should be 462 KB -see attached .zip-).

#### Screenshots
![Screenshot 2024-05-16 212134](https://github.com/tophf/mpiv/assets/723651/a914eec2-598f-49b5-a79c-16acc4a24c4d)

#### Environment
 - Browser: [Chrome, Firefox]
Chrome 125.0.6422.61

 - MPIV Version:
1.3.0

I attach [both downloaded files](https://github.com/tophf/mpiv/files/15339594/the.2.downloaded.files.zip)

**Comment by tophf** [comment]
Sounds like a problem in Tampermonkey beta, not sure what I can do about it...

**Comment by tophf** [comment]
By the looks of it Tampermonkey incorrectly encodes the response as plain text.

**Comment by darkred** [comment]
I'm glad this issue is fixed in latest Tampermonkey beta 5.2.6197. Thanks for reporting this to https://github.com/Tampermonkey/tampermonkey/issues/2064.
(the issue didn't occur with Firefox in my report in the OP because at that time its version was an earlier one, 5.2.6194)
